cpp/include/cugraph_c/graph.h

```diff
@@ -116,19 +163,67 @@ cugraph_error_code_t cugraph_sg_graph_create_from_csr(
   cugraph_graph_t** graph,
   cugraph_error_t** error);
 
+/**
+ * @brief     Construct an SG graph from a CSR input
+ *
+ * @param [in]  handle         Handle for accessing resources
+ * @param [in]  properties     Properties of the constructed graph
+ * @param [in]  offsets        Device array containing the CSR offsets array
+ * @param [in]  indices        Device array containing the destination vertex ids
+ * @param [in]  weights        Device array containing the edge weights.  Note that an unweighted
+ *                             graph can be created by passing weights == NULL.
+ * @param [in]  edge_ids       Device array containing the edge ids for each edge.  Optional
+                               argument that can be NULL if edge ids are not used.
+ * @param [in]  edge_type_ids  Device array containing the edge types for each edge.  Optional
+                               argument that can be NULL if edge types are not used.
+ * @param [in]  store_transposed If true create the graph initially in transposed format
+ * @param [in]  renumber       If true, renumber vertices to make an efficient data structure.
```

Comment:
How does `renumber=True` work for CSR? Wouldn't `offsets` already imply 0..N-1 vertex ids?

Reply:
`renumber=True` would still work for CSR input.  Yes, `offsets` does imply vertex ids in the range `[0, N)`.  But our renumbering logic adds some performance optimizations by putting the vertex ids in an order that makes the CUDA kernels more efficient.  So there is value in specifying `renumber=True` even if the vertex ids are densely packed.

I will be fixing the C API implementation of this function so that it will properly reflect the isolated vertices that can be implied in CSR format.